frontend/src/index.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
If you look a couple of lines below you will see that the min file has already been imported.